<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LDBB7HE" failures="3" tests="6" name="CartTest" time="37.469" errors="0" timestamp="2024-01-01T19:31:30 EET">
  <testcase classname="TestPackage.cartTestcases" name="checkOutTheorder" time="19.912"/>
  <testcase classname="TestPackage.cartTestcases" name="fillCheckoutDetials" time="2.133">
    <failure type="java.lang.AssertionError" message="expected [Error: Posta Code is required] but found [Error: Postal Code is required]">
      <![CDATA[java.lang.AssertionError: expected [Error: Posta Code is required] but found [Error: Postal Code is required]
at TestPackage.cartTestcases.fillCheckoutDetials(cartTestcases.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- fillCheckoutDetials -->
  <testcase classname="TestPackage.cartTestcases" name="removeIteamFromcart" time="5.912">
    <failure type="java.lang.AssertionError" message="expected [2] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [0]
at TestPackage.cartTestcases.removeIteamFromcart(cartTestcases.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- removeIteamFromcart -->
  <testcase classname="TestPackage.cartTestcases" name="verifyCartPageOpened" time="0.964"/>
  <testcase classname="TestPackage.cartTestcases" name="verifyContinueShoppingButton" time="0.996"/>
  <testcase classname="TestPackage.cartTestcases" name="verifyProductPriceValidDuringCheckoutProcess" time="0.234">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index 2 out of bounds for length 2">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index 2 out of bounds for length 2
at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:266)
at java.base/java.util.Objects.checkIndex(Objects.java:359)
at java.base/java.util.ArrayList.get(ArrayList.java:427)
at PageObjects.ProductsPage.getRandomProductPrice(ProductsPage.java:41)
at TestPackage.cartTestcases.verifyProductPriceValidDuringCheckoutProcess(cartTestcases.java:140)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- verifyProductPriceValidDuringCheckoutProcess -->
</testsuite> <!-- CartTest -->
